src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Formatting/FormattingPassBase.cs

```diff
@@ -168,15 +178,66 @@ protected async Task<List<TextChange>> AdjustIndentationAsync(FormattingContext
                     continue;
                 }
 
-                var startIndentation = await CSharpFormatter.GetCSharpIndentationAsync(context, mapping.GeneratedSpan.AbsoluteIndex, cancellationToken);
-                sourceMappingIndentations[mapping.OriginalSpan.AbsoluteIndex] = startIndentation;
+                var startLocation = mapping.GeneratedSpan.AbsoluteIndex;
+                sourceMappingMap[mapping.OriginalSpan.AbsoluteIndex] = startLocation;
+                significantLocations.Add(startLocation);
+
+                var endLocation = mapping.GeneratedSpan.AbsoluteIndex + mapping.GeneratedSpan.Length + 1;
+                sourceMappingMap[mapping.OriginalSpan.AbsoluteIndex + mapping.OriginalSpan.Length + 1] = endLocation;
```

Comment:
Nit; extract `mapping.OriginalSpan.AbsoluteIndex + mapping.OriginalSpan.Length + 1` to var